table/column.go

```diff
@@ -362,6 +362,9 @@ func CheckNotNull(cols []*Column, row []types.Datum) error {
 
 // GetColOriginDefaultValue gets default value of the column from original default value.
 func GetColOriginDefaultValue(ctx sessionctx.Context, col *model.ColumnInfo) (types.Datum, error) {
+	if col.OriginDefaultValue != nil && col.DefaultValueBit != nil && col.Tp == mysql.TypeBit {
```

Comment:
Add a comment here will be illustrative.

Reply:
Prefer placing the `col.Tp == mysql.TypeBit` ahead of other conditions (I think this has better filtering to avoid unnecessary compare, e.g: `col.OriginDefaultValue != nil && col.DefaultValueBit != nil`).

Reply:
@lonng Updated. PTAL